packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/add-user/add-user.component.js

```diff
@@ -13,6 +13,7 @@ export default {
     objectKey: '<',
     goBack: '<',
     goToUsersAndRoles: '<',
-    trackingPrefix: '<',
+    trackClick: '<',
+    trackPage: '<',
```

Comment:
**nitpick:** Not used anymore
```suggestion
```

Reply:
It's used in the function `trackAddUserModalPage()` & `trackAddUserModalClick()` of the controller.